web/html/src/manager/content-management/shared/components/panels/properties/properties-view.js

```diff
@@ -33,22 +55,29 @@ const PropertiesView= (props: Props) => {
         <dl className="row">
           <dt className="col-xs-2">Versions history:</dt>
```

Comment:
`t()` function missing here as well and in all the previous field. I see this is something detached from this PR but maybe it's a good occasion to fix them all. :wink: